README.rst

```diff
@@ -59,7 +59,7 @@ You can render an Airflow Task Group using the ``DbtTaskGroup`` class. Here's an
         dbt_tg = DbtTaskGroup(
             dbt_project_name="jaffle_shop",
             conn_id="airflow_db",
-            dbt_args={
+            profile_args={
```

Comment:
Good question, I left it as `profile_args` because these are explicitly placed into the dbt profile. In vanilla dbt you can define multiple profiles, just as with Airflow you can have multiple connections. But you'd typically only use one per dbt project, just as you'd typically only use Airflow connection per operator. 

Before 1.0 we should also look at the params a user has to pass into the cosmos objects and make sure they make sense. Lumping everything under a single untyped `dbt_args` doesn't make a whole lot of sense to me because the arguments are used in different ways. This is just the first opportunity to do that.

Reply:
If you feel strongly that this should be `profile_target_args` I'd be open to that, but I don't think it should cause that much confusion as-is because you only specify a single target/profile. We don't really have "single profile with many targets" like in vanilla dbt.